### DISCUSSION TITLE
SenseCAP K1100 Guide - The Sensor Prototype Kit | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1367] [Q&A]
# SenseCAP K1100 Guide - The Sensor Prototype Kit | Seeed Studio Wiki

SenseCAP K1100 Guide - The Sensor Prototype Kit

https://wiki.seeedstudio.com/SenseCAP_K1100_Intro/

**Comment by eerke2021** [comment]
Can I convert a K1100 sensor dev kit to work with Meshtastic? I see the peer to peer project above. Maybe I need to flash the firmware on the Grove E5 itself?
